.github/workflows/check_pr.yaml

```diff
@@ -1,7 +1,10 @@
-name: Check Pull Request
+name: Tests
 run-name: ${{ github.actor }} checking pull request
 
 on:
+  push:
+    branches:
+      - main
```

Comment:
this is so it will run on main branch when we merge so that the badge will update

Reply:
Is this the second badge you are referring to in the Changelog?

Reply:
yep theres a built in syntax for adding badges for GA workflows, no need to generate an svg like for interrogate which is much nicer, as currently the svg is generated on commit so you then have to commit again to add it.... not great so maybe we change this in future